README.md

```diff
@@ -1,86 +1,173 @@
+...
+
+▶ sudo atomicapp run --provider=kubernetes .
+...
+
+▶ # or
+
+▶ docker build -t myapp
+...
+▶ sudo atomicapp run --provider=kubernetes myapp
+...
 ```
 
-Use 'docker build' to package up the application and tag the resulting image.
+## Real-world examples
+Atomic App can be used to launch a cluster of containers (application servers, databases, etc.).
 
-### Fetch and run
-```
-atomicapp [--dry-run] [-v] [-a answers.conf] fetch|run|stop|genanswers [--provider docker] [--destination DST_PATH] APP|PATH
-```
+For a list of already packaged examples, visit the [nulecule-library](https://github.com/projectatomic/nulecule-library) repo.
 
-Pulls the application and its dependencies. If the last argument is
-existing path, it looks for `Nulecule` file there instead of pulling anything.
+## Contributing to Atomic App
+[![Code Health](https://landscape.io/github/projectatomic/atomicapp/master/landscape.svg?style=flat)](https://landscape.io/github/projectatomic/atomicapp/master)
+[![Build Status](https://travis-ci.org/projectatomic/atomicapp.svg?branch=master)](https://travis-ci.org/projectatomic/atomicapp)
+[![Coverage Status](https://coveralls.io/repos/projectatomic/atomicapp/badge.svg?branch=master&service=github)](https://coveralls.io/github/projectatomic/atomicapp?branch=master)
+[![Issue Stats](http://issuestats.com/github/projectatomic/atomicapp/badge/pr)](http://issuestats.com/github/projectatomic/atomicapp)
+[![Issue Stats](http://issuestats.com/github/projectatomic/atomicapp/badge/issue)](http://issuestats.com/github/projectatomic/atomicapp)
 
-* `--provider docker` Use the Docker provider within the Atomic App
-* `--destination DST_PATH` Unpack the application into given directory instead of current directory
-* `APP` Name of the image containing the application (ex. `projectatomic/apache-centos7-atomicapp`)
-* `PATH` Path to a directory with installed (ex. result of `atomicapp fetch...`) app
+First of all, awesome! We have [a development guide to help you get started!](CONTRIBUTING.md)
 
-Action `run` performs `fetch` prior to its own tasks if an `APP` is provided. Otherwise, it will use its respective `PATH`. When `run` is selected, providers' code is invoked and containers are deployed.
+If you have any issues or get stuck, feel free to open a GitHub issue or reach us at our communication channels (see below).
 
 ## Providers
 
-Providers represent various deployment targets. They can be added by placing the artifact within the respective in `provider/` folder. For example, placing `deploy_pod.yml` within `providers/kubernetes/`. For a detailed description of all providers available see [docs/providers.md](docs/providers.md).
+We currently support:
+
+  - Docker
+  - Kubernetes
+  - OpenShift 3
+  - Marathon (Mesos)
+
+Providers represent various deployment targets. They can be added by placing the artifact within the respective in `provider/` folder. For example, placing `deploy_pod.yml` within `providers/kubernetes/`. 
```

Comment:
The examples in nulecule-library have the providers directly in the artifacts directory.  Where is this provider/ directory supposed to be?